### ISSUE TITLE
How can I add chapters to my mission?

### ISSUE THREAD

**Issue by olegfedorovdev** [ctrlaltninja/druidstone_modding#1]
Hello,

Documentation does not say how to add new chapters. Can you add that please?

**Comment by olegfedorovdev** [comment]
The biggest help for modders would be ability to look into your druidstone.dat - it has all missions so we could copy and learn from there

**Comment by Larr-Dev** [comment]
Even if you arent modding still, someone someday may need a guide =) I look forward to seeing an updte to druidstone chess!

<code>
def_mission{

		name = "$mod.first_mission", 
		site = "$mod.globally_defined_site",
		ui_name = "Surface",
		level_filename = "mod_data/levels/surface.level",
		letter_text = "From: XXX\n" ..
		"To: YYY\n" ..
		"Subject: Druidstone modders are being stored underground.\n\n" ..
		"Thats why no mew mods!!!",
		next_mission = "$mod.second_mission",
		locked = false,
	}
</code>
<code>
def_mission{
	
	name = "$mod.second_mission",
	site = "$mod.globally_defined_site",
	ui_name = "Modders Prison",
	level_filename = "mod_data/levels/modders_prison.level", 
      next_mission = "you get it, right?",
	locked = false,
	}

</code>

**Comment by Larr-Dev** [comment]
I agree with the .dat ... it looks like LoG 2 modders unpacked it overtime with python.  I am not qualified to do that but if anyone opens it up, id like to see some implementation with the ability functions. choose_target for instance, execute_attack... 

/onward!

**Comment by Larr-Dev** [comment]
There must be a mission flag that declares an Interlude also .  Probably a
flags = "" field. I haven't been able to figure it out as there is no hints as to what it may be.

**Comment by olegfedorovdev** [comment]
Thank you
